CMakeLists.txt

```diff
@@ -702,6 +702,10 @@ if(CONFIG_APPLICATION_MEMORY)
   list(APPEND LINKER_SCRIPT_DEFINES ${def})
 endif() # CONFIG_APPLICATION_MEMORY
 
+if(CONFIG_ARM)
```

Comment:
How is this specific to ARM?

Reply:
Not specific to ARM, as of now it is being tested for ARM only...since this feature more time to test, initial implementation is being done and tested for ARM only, later other arch will be also be supported.